http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/Boot.scala

```diff
@@ -869,6 +869,7 @@ object Boot extends IOApp {
         .withConnectTimeout(30 seconds)
         .withRequestTimeout(60 seconds)
         .withMaxTotalConnections(100)
+        .withMaxWaitQueueLimit(1024)
```

Comment:
This controls the size of the queue of connections (which is constrained by `withMaxTotalConnections` for the number that can be open concurrently). Before it was 120 -- I think it makes sense for the queue to be a bit larger than `withMaxTotalConnections`.